docs/overview/component/index.md

```diff
@@ -4,17 +4,6 @@ This document will give you an overview of all the components developed and used
 
 ## Table of Contents <!-- omit in toc -->
 
-- [Vald Filter](#vald-filter)
-  - [Vald Ingress Filter](#vald-ingress-filter)
-  - [Vald Egress Filter](#vald-egress-filter)
-  - [Vald Filter Gateway](#vald-filter-gateway)
```

Comment:
`vald filter` and `vald filter gateway` is provided right?

Reply:
restored :) 
I was my mistake, thank you